### PR TITLE
Changes to frontend dependencies

### PR DESCRIPTION
**Pull request by armenzg**
I'm re-evaluating what packages we still need and moving some to devDependencies.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7244,7 +7244,7 @@ pn@^1.1.0:
   resolved "https://registry.yarnpkg.com/pn/-/pn-1.1.0.tgz#e2f4cef0e219f463c179ab37463e4e1ecdccbafb"
   integrity sha512-2qHaIQr2VLRFoxe2nASzsV6ef4yOOH+Fi9FBOVH6cqeSgUnoyySPZkxzLuzd+RYOQTRpROA0ztTMqxROKSb/nA==
 
-popper.js@1.15.0, popper.js@^1.14.1:
+popper.js@^1.14.1:
```
Comment (sarah-clements) [inline yarn.lock]: I'm curious this was changed.

File: package.json [hunk 2]
```diff
@@ -47,10 +44,8 @@
     "metrics-graphics": "2.15.6",
     "moment": "2.24.0",
     "mousetrap": "1.6.3",
-    "neutrino": "9.0.0-rc.3",
     "ng-text-truncate-2": "1.0.1",
     "numeral": "2.0.6",
-    "popper.js": "1.15.0",
```
Comment (sarah-clements) [inline package.json]: I think popper is used by react-strap. Have you tested dropdowns and tooltips to make sure they still work?

Comment (armenzg) [reply package.json]: Yes, it is needed, however, it is not needed as a peer dependency.

Before this change we were installing `popper.js` twice. Once as a peer-dependency and another one as part of `reactstrap`'s dependencies.

This dropdown seems to work:
image

This tooltip seems to work:
image

This also seems to work:
image

```shell
Armens-MacBook-Pro:treeherder armenzg$ npm ls popper.js
treeherder@ /Users/armenzg/repos/treeherder
└─┬ reactstrap@7.1.0
  └─┬ react-popper@0.10.4
    └── popper.js@1.15.0
```

Comment (sarah-clements) [reply package.json]: I see - makes sense.

**Pull request merged**